### DISCUSSION TITLE
swaymsg -t get_tree?

### DISCUSSION THREAD

**Discussion by bobblkabb** [hyprwm/Hyprland#741] [Q&A] [answered]
Command doesn't work, getting error: 
`00:00:00.046 [swaymsg/main.c:419] Unable to retrieve socket path`

How come? Is this hyprland specific? Is it my distro (openSUSE Tumbleweed)? Is there an alternative?

**Comment by fufexan** [comment, marked as answer]
How would swaymsg work outside of Sway?

We have our own control binary, [hyprctl](https://wiki.hyprland.org/Configuring/Using-hyprctl/)

**Reply by iamtvk** [reply]
how can I perform 
`swaymsg "output * dpms off `
with  hyprctl

**Reply by vaxerski** [reply]
`hyprctl dispatch dpms off`

**Comment by bobblkabb** [comment]
Thank you. Totally missed it in the wiki ('Window Rules'). Sorry
